queries.sql

```diff
@@ -9,3 +9,9 @@ SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
+SELECT neutered, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY neutered;
+SELECT neutered, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1-1-1990' AND '1-1-2000' GROUP BY neutered;
```

Comment:
- [ ] Well done running the queries but I double-checked that the screenshots you shared in the last part of your PR are not the result of these queries. Although these queries are not wrong and may fulfill the requirements, the results screenshots should be in the PR. kindly consider fixing it.